apollo-router/src/error.rs

```diff
@@ -132,6 +144,26 @@ impl FetchError {
     }
 }
 
+impl ErrorExtension for FetchError {
+    fn extension_code(&self) -> String {
+        match self {
+            FetchError::ValidationInvalidTypeVariable { .. } => "ValidationInvalidTypeVariable",
+            FetchError::ValidationPlanningError { .. } => "ValidationPlanningError",
+            FetchError::SubrequestMalformedResponse { .. } => "SubrequestMalformedResponse",
+            FetchError::SubrequestUnexpectedPatchResponse { .. } => {
+                "SubrequestUnexpectedPatchResponse"
+            }
+            FetchError::SubrequestHttpError { .. } => "SubrequestHttpError",
+            FetchError::ExecutionFieldNotFound { .. } => "ExecutionFieldNotFound",
+            FetchError::ExecutionPathNotFound { .. } => "ExecutionPathNotFound",
+            FetchError::CompressionError { .. } => "CompressionError",
+            #[cfg(test)]
+            FetchError::ExecutionInvalidContent { .. } => "ExecutionInvalidContent",
+        }
+        .to_shouty_snake_case()
```

Comment:
Can we not use this and instead just make the string constants shouty snake case? That way if we have to search for these constants we'll find them. Same goes for all other places where this is used.